src/eoc.js

```diff
@@ -69,10 +69,12 @@ program.command('foreign')
     foreign(program.opts());
   });
 
-program.command('clean')
+program
+  .command('clean')
+  .option('--cached', 'delete ~/.eo directory')
   .description('delete all temporary files')
   .action((str, opts) => {
-    clean(program.opts());
+    clean({...str, ...program.opts()});
```

Comment:
@maxonfjvipon I have a strong feeling that it's wrong. Try to run this and you will the problem:

```
$ eoc clean --cached --foo
```

Reply:
with `--foo` we get `error: unknown option '--foo'`, so it works absolutely correctly, but not quite obviously, will fix it.

Reply:
fixed